interpretation/deepseismic_interpretation/dutchf3/data.py

```diff
@@ -576,7 +656,18 @@ def transform(self, img, lbl):
 
 
 class TestPatchLoader(PatchLoader):
-    def __init__(self, data_dir, stride=30, patch_size=99, is_transform=True, augmentations=None):
+    """
+    Test Data loader for the patch-based deconvnet
+    :param str data_dir: Root directory for training/test data
+    :param int stride: training data stride
+    :param int patch_size: Size of patch for training
+    :param bool is_transform: Transform patch to dimensions expected by PyTorch
+    :param list augmentations: Data augmentations to apply to patches
+    :param str seismic_path: Override file path for seismic data
+    :param str label_path: Override file path for label data
```

Comment:
these parameters don't exist in the __init__(). Can they be removed?

Reply:
@yalaudah I can do this. :-) However, we have to close this PR and reopen a new one pointing out to the new contrib branch.